src/content/structured/components/chips/guidance.mdx

```diff
@@ -33,28 +33,28 @@ import chipsVid2 from "./images/anim-2-do-not-use-dismissible-chips-when-the-use
 ## Component variants
 
 <ComponentPreview style={{ gap: "8px" }}>
-  <IcChip label="Default" dismissible>
+  <IcChip label="Americano" dismissible>
     <svg
       slot="icon"
-      width="20"
-      height="20"
-      viewBox="0 0 20 20"
+      width="26"
+      height="26"
+      viewBox="0 0 26 26"
       fill="currentColor"
       xmlns="http://www.w3.org/2000/svg"
     >
-      <path d="M10 0C4.48 0 0 4.48 0 10C0 15.52 4.48 20 10 20C15.52 20 20 15.52 20 10C20 4.48 15.52 0 10 0ZM10 3C11.66 3 13 4.34 13 6C13 7.66 11.66 9 10 9C8.34 9 7 7.66 7 6C7 4.34 8.34 3 10 3ZM10 17.2C7.5 17.2 5.29 15.92 4 13.98C4.03 11.99 8 10.9 10 10.9C11.99 10.9 15.97 11.99 16 13.98C14.71 15.92 12.5 17.2 10 17.2Z" />
+      <path d="M2,21V19H20V21H2M20,8V5H18V8H20M20,3A2,2 0 0,1 22,5V8A2,2 0 0,1 20,10H18V13A4,4 0 0,1 14,17H8A4,4 0 0,1 4,13V3H20M16,5H6V13A2,2 0 0,0 8,15H14A2,2 0 0,0 16,13V5Z" />
     </svg>
   </IcChip>
-  <IcChip label="Default">
+  <IcChip label="Cappuccino">
     <svg
       slot="icon"
-      width="20"
-      height="20"
-      viewBox="0 0 20 20"
+      width="26"
+      height="26"
+      viewBox="0 0 26 26"
```

Comment:
I think we can remove the width and height values entirely because they could be messing with the ones we have set on the chip icon. And then if we update the viewBox to a regular 24px one then I think it looks correct with the padding values etc.
```suggestion
      viewBox="0 0 24 24"
```